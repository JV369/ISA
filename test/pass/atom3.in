./feedreader http://www.mocky.io/v2/5bf2b4362f00007326cfa225 -au
