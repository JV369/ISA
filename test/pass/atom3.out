*** Test ***
Test entry
Autor: JV
URL: http://hue.org

Test entry 2
Autor: JÃ¶rvearick
URL: http://hue.com

Test ex entry 1
Autor: Source
URL: http://euh.com

Test ex entry 2
Autor: TestMissing
URL: http://right.com

